### PR TITLE
Adding /base64 support

### PR DESCRIPTION
**Pull request by saurabh-hirani**
* Adding as per httpbin.org implementation.

### CODE REVIEW

**Comment by saurabh-hirani** [comment]
@mccutchen  - although adherence to httpbin.org helps - I wanted check if we could do one better by having -

```
/base64/:value - decode and return :value
/base64/encode/:value - encode and return value
/base64/decode/:value - decode and return value
```

thereby making encoding and decoding explicit, pipeable as well as supporting httpbin.org as is. Let me know your thoughts so that if required, I will implement the /base64/encode/:value and /base64/:decode/value endpoints.

**Comment by mccutchen** [comment]
@saurabh-hirani ooh yeah, I think your proposed improvements make sense!

File: httpbin/handlers_test.go [hunk 1]
```diff
@@ -2190,6 +2190,36 @@ func TestUUID(t *testing.T) {
...
+	assertStatusCode(t, w, http.StatusOK)
+	expectedContent := "HTTPBINGO is awesome"
+	gotContent := w.Body.String()
+
+	if gotContent != expectedContent {
+		t.Fatalf("Expected [%s] - Got [%s]", expectedContent, w.Body.String())
+	}
+}
+
+func TestInvalidBase64(t *testing.T) {
```
Comment (mccutchen) [inline httpbin/handlers_test.go]: One thing I'd like to move towards in these tests, where it's not too much of a burden, is defining a single top-level `TestBase64` function for an endpoint and defining individual cases within it using `t.Run()`.

Comment (saurabh-hirani) [reply httpbin/handlers_test.go]: Ok - will check and update in a few days.

Comment (saurabh-hirani) [reply httpbin/handlers_test.go]: Done.

**Comment by saurabh-hirani** [comment]
@mccutchen  - incorporated the feedback. When I squashed my commits - after adding /base64 to index.html - I created the above conflict by mistake on assets.go and index.html. Sorry for that - It should be resolveable by updating index.html with https://github.com/saurabh-hirani/go-httpbin/blob/add-base64-support/static/index.html#L108#L111 - rest all is new code for base64 support as per feedback. 

Can you see if it can be resolved easily? If it can't be - I am fine with deleting this branch and creating a new one and submitting a new PR with the above changes. Thanks.

**Comment by saurabh-hirani** [comment]
@mccutchen  - Done as per feedback for multiple base64 endpoints and subtests requirement. Ran into a merge conflict as I had not pulled your /uuid index.html changes. Fixed that and ran "make assets" to confirm.

File: httpbin/helpers.go [hunk 2]
```diff
@@ -243,3 +249,70 @@ func uuidv4() (string, error) {
...
+}
+
+// Decode - decode data from base64
+func (b *base64Helper) Decode() ([]byte, error) {
+	buff := make([]byte, base64.StdEncoding.DecodedLen(len(b.data)))
+	_, err := base64.StdEncoding.Decode(buff, []byte(b.data))
+	return buff, err
+}
+
+func randStringBytes(n int) string {
```
[the remark this answers is not in the document]
Comment (saurabh-hirani) [reply httpbin/helpers.go]: Done.

File: static/index.html [hunk 3]
```diff
@@ -105,6 +105,9 @@ <h2 id="ENDPOINTS">ENDPOINTS</h2>
 <li><a href="/image/svg"><code>/image/svg</code></a> Returns a SVG image.</li>
 <li><a href="/forms/post" data-bare-link="true"><code>/forms/post</code></a> HTML form that submits to <em>/post</em></li>
 <li><a href="/uuid"><code>/uuid</code></a> Generates a <a href="https://en.wikipedia.org/wiki/Universally_unique_identifier">UUIDv4</a> value.</li>
+<li><a href="/base64/:value"><code>/base64/:value</code></a> Decodes a Base64 encoded string.</li>
+<li><a href="/base64/decode/:value"><code>/base64/decode/:value</code></a> Explicit URL for decoding a Base64 encoded string.</li>
+<li><a href="/base64/encode/:value"><code>/base64/encode/:value</code></a> Encodes a string into Base64.</li>
```
[the remark this answers is not in the document]
Comment (saurabh-hirani) [reply static/index.html]: Makes sense - done.

**Pull request merged**